akka-http-core/src/main/resources/reference.conf

```
@@ -278,6 +278,14 @@ akka.http {
 
       # Enable verbose debug logging for all ingoing and outgoing frames
       log-frames = false
+
+      # When there is no data transmitted in either direction, but there are active streams, send a HTTP/2 ping frame with this initial
+      # delay and subsequent interval to make sure the connection is kept alive, 0s disables sending ping frames. Must be full seconds.
+      ping-interval = 0s
+
+      # Fail the connection if a ping frame is sent and does not come back from the server within this timeout.
```

Maybe slightly simpler (also below for client):

```suggestion
      # Fail the connection if a sent ping is not acknowledged within this timeout.
```